pkg/relay/register.go

```diff
@@ -0,0 +1,250 @@
+package relay
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/blocknative/dreamboat/pkg/structs"
+	"github.com/flashbots/go-boost-utils/types"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	ResponseTypeVerify = iota
+	ResponseTypeOthers
+	ResponseTypeStored
+)
+
+type Setter interface {
+	Set(k string, value uint64)
+}
+
+type Getter interface {
+	Get(k string) (value uint64, ok bool)
+}
+
+// VSReq is a request structure used in communication
+// between api calls and fixed set of worker goroutines
+// it's using return channel pattern, meaning that after
+// sent the sender locks on that channel to get the response
+type VSReq struct {
```

Comment:
What does VS stand for?

Reply:
Yeah I changed that already after Aratz's comment to VerifyReq :)